src/services/ConsultDialogService.ts

```diff
@@ -1,15 +1,20 @@
-const Dialog = require("../models/Dialog");
-const Response = require("../utils/responses");
+import { Request, Response } from "express";
 
-module.exports = async (req, res) => {
+import { ResponseHandler } from "../utils/ResponseHandler";
+import Dialog from "../models/Dialog";
+
+export default async function ConsultDialogService(
+  req: Request,
+  res: Response
+) {
   const { limit, page } = req.headers;
 
   const options = {
-    limit: parseInt(limit, 10) || 10,
-    page: parseInt(page, 10) || 1,
+    limit: parseInt(limit as string, 10) || 10,
```

Comment:
`limit` and `page` must be numbers.

Or do you really think it should be a `string`? If so, why?